### PR TITLE
Install vfsgen for operator repo

### PR DESCRIPTION
**Pull request by sdake**
vfsgen produces an output binary and is necessary such that mac
builds do not overwrite the Linux architecture binaries.

This tool is used by the operator repository.

### CODE REVIEW

File: docker/build-tools/Dockerfile [hunk 1]
```diff
@@ -102,4 +109,11 @@ COPY --from=golang_context /usr/local/go/ /go/
 RUN mkdir -p /gocache
 RUN mkdir -p /go/pkg/mod
 
+# TODO must sort out how to use uid mapping in docker so these don't need to be 777
```
Comment (sdake) [inline docker/build-tools/Dockerfile]: hmm not sure where this emanated from.

Comment (sdake) [reply docker/build-tools/Dockerfile]: let me delete this and rebuild before merging please.

File: docker/build-tools/Dockerfile [hunk 2]
```diff
@@ -53,9 +54,15 @@ RUN GO111MODULE=on go get golang.org/x/tools/cmd/goimports@${GOIMPORTS_VERSION}
 RUN GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_LINT_VERSION}
 RUN go get github.com/jteeuwen/go-bindata/go-bindata
 
-# Install latest versions of custom Istio tools
+
+# Install per-repo binaries
+# Anything that is installed for use with go generate needs to be installed
+# here for correct operation on MacOS and Windows platforms.
+## https://github.com/istio/istio.io
```
Comment (geeknoid) [inline docker/build-tools/Dockerfile]: These are used by the istio/api repo.

Comment (geeknoid) [reply docker/build-tools/Dockerfile]: I don't think it's just about mac/windows. It's about having the build tools in the build container image in general. That's the point of the container image.

Comment (sdake) [reply docker/build-tools/Dockerfile]: will address.

**Pull request merged**